platforms/hyperledger-fabric/configuration/molecule/pre-vault- setup/cleanup.yml

```diff
@@ -0,0 +1,32 @@
+---
+  - name: Cleanup
```

Comment:
folder name is wrong. file alignment is wrong.

Reply:
Corrected